integration/spark/app/src/test/java/io/openlineage/spark/agent/SparkGenericIntegrationTest.java

```diff
@@ -193,6 +193,24 @@ void sparkEmitsEnvironmentPropertiesFacet() {
             });
   }
 
+  @Test
+  void sparkEmitsJobTypeJobFacet() {
+    Dataset<Row> df = createTempDataset();
+
+    Dataset<Row> agg = df.groupBy("a").count();
+    agg.write().mode("overwrite").csv("/tmp/test_data/test_output/");
+    spark.stop();
+
+    List<OpenLineage.RunEvent> events = getEventsEmitted(mockServer);
+
+    // Both Spark application and Spark job events have jobType facet
+    assertThat(events)
```

Comment:
Would it make sense to include this assertion in the test below `sparkEmitsApplicationDetailsFacet`?

Reply:
Do you mean move this assertion to `sparkEmitsApplicationDetailsFacet`?

Reply:
yes, move current assertion to `sparkEmitsApplicationDetailsFacet` and remove current test.

Reply:
Ok, merged most of these tests into one test with multiple asserts